examples/requirements.txt

```
@@ -5,7 +5,7 @@ functorch
 
 --requirement ../requirements.txt
 
-gym < 1.0.0.a0
+gym == 0.15.3
```

I developed my examples based on this version of gym, don't trust other versions.

---

`gym 0.15.3` is too old.

```suggestion
gym >= 0.20.0, < 0.24.0a0
```